test/e2e/healthcheck_run_test.go

```diff
@@ -52,6 +52,18 @@ var _ = Describe("Podman healthcheck run", func() {
 		Expect(hc).Should(Exit(125))
 	})
 
+	FIt("podman healthcheck from image's config (not container config)", func() {
```

Comment:
are you testing just this test or does it need to be changed?

Reply:
That needs to be changed. Thank you for catching!